25-multiplecomponents-lab_v16/src/01-redspotter.jsx

```diff
@@ -1,13 +1,58 @@
-/**
- * Lab 01:
- *
- * Write a game that shows 10 squares: 9 gray and 1 red.
- * When a user clicks on the red she gets 5 points and red changes its place.
- * When a user clicks on any other square nothing happens.
- *
- */
-
-import React from 'react';
-
-export default class RedSpotter extends React.Component {
-}
+import React from 'react';
+
+export default class RedSpotter extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            score: 0,
+            boxes: [
+                { id: 10, color: "#ff0000" },
+                { id: 20, color: "#c0c0c0" },
+                { id: 30, color: "#c0c0c0" },
+                { id: 40, color: "#c0c0c0" },
+                { id: 50, color: "#c0c0c0" },
+                { id: 60, color: "#c0c0c0" },
+                { id: 70, color: "#c0c0c0" },
+                { id: 80, color: "#c0c0c0" },
+                { id: 90, color: "#c0c0c0" },
+                { id: 100, color: "#c0c0c0" }
+            ]
+        };
+    }
+
+    shuffle = (id) => {
+        var _ = require('lodash');

```

Comment:
1. Better to `require` all libraries once at the top of the file
2. Better to use `import`